package.json

```diff
@@ -3,35 +3,43 @@
   "version": "0.2.0",
   "dependencies": {
     "body-parser": "~1.0.2",
-    "cors": "~2.2.0",
-    "express": "~4.0.0",
-    "request": "2.79.0",
-    "fh-mbaas-api": "~7.0.9"
+    "cors": "~2.8.3",
+    "express": "~4.15.3",
+    "fh-mbaas-api": "~7.0.11",
+    "grunt": "^1.0.1",
+    "request": "2.81.0"
   },
   "devDependencies": {
     "grunt-concurrent": "latest",
-    "grunt-contrib-jshint": "^0.11.3",
+    "grunt-contrib-jshint": "^1.1.0",
     "grunt-contrib-watch": "latest",
     "grunt-env": "~0.4.1",
     "grunt-node-inspector": ">=0.2.0",
     "grunt-nodemon": "0.4.2",
     "grunt-open": "~0.2.3",
-    "grunt-plato": "~1.0.0",
-    "grunt-shell": "^0.7.0",
-    "istanbul": "0.2.7",
-    "load-grunt-tasks": "^0.4.0",
-    "mocha": "^2.1.0",
-    "proxyquire": "0.5.3",
-    "should": "2.1.1",
-    "sinon": "^1.17.2",
-    "supertest": "0.8.2",
-    "time-grunt": "^0.3.2"
+    "grunt-plato": "~1.4.0",
+    "grunt-shell": "^2.1.0",
```

Comment:
Ya you're right my bad. They won't be backward compatible with node 4. I'll change them back thanks